pkg/gadgets/trace/exec/tracer/bpf/execsnoop.bpf.c

```diff
@@ -220,8 +207,42 @@ int ig_execve_x(struct syscall_trace_exit *ctx)
 	if (len <= sizeof(*event))
 		bpf_perf_event_output(ctx, &events, BPF_F_CURRENT_CPU, event,
 				      len);
+
+	return 0;
+}
+
+// We use syscalls/sys_exit_execve only to trace failed execve
```

Comment:
nit: comment that this program is needed in all cases, not only when ignore-errors == false. (It was my first impression at the beginning when looking at the PR)